Source/Core/Color.js

```diff
@@ -474,7 +474,7 @@ import CesiumMath from './Math.js';
      * @returns {Number} The converted number.
      */
     Color.floatToByte = function(number) {
-        return number === 1.0 ? 255.0 : (number * 256.0) | 0;
+        return Math.round(number * 255.0);
```

Comment:
The `| 0` was removed from the end of the formula, is that OK?  I know JS doesn't make a strong distinction between integers and floating-point, but I think the `| 0` at the end was being used for some under-the-hood coercion.  At the native code level, `round` returns a double-precision floating-point number, but `|` is a bitwise operator that returns an integer.

Reply:
I can add the "integerization" back in, but the concern seems misplaced to me.

Reply:
Yeah I haven't performance tested, and I'm not familiar with what's using this code.  I'm just asking the question.

Reply:
I've created another branch with the bitwise integerization if you want to test; I'm curious the results.
https://github.com/Aviture/cesium/tree/fix-color-float-to-byte-bitwise-integerize

Reply:
`| 0` is a truncation, basically using bitwise operations as a faster alternative to Math.floor.